nodejs/week1/mealSharing/src/backend/routes/meals.js

```diff
@@ -0,0 +1,14 @@
+const mealsData = require("../data/meals.json");
+const reviewsData = require("../data/reviews.json");
+const mealsFunc = function (request, response) {
+    mealsData.forEach((mealItem) => {
+        mealItem.review = [];    
+        reviewsData.forEach((reviewItem) => {
+           if (reviewItem.mealId === mealItem.id) {
+              mealItem.review.push(reviewItem);
+           };
+        });
+     });
```

Comment:
Seems to be working but using `map` and `filter` would in my opinion be a bit simpler.